### PR TITLE
[vcpkg baseline] Fix libxmu dep

### PR DESCRIPTION
**Pull request by Neumann-A**
missed libxext as a dependency. Thought it would be added through libxt

### CODE REVIEW

File: versions/l-/libxmu.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "59f551a642f1c8b1c6dd02ad3f13d32231e48d4b",
+      "git-tree": "6bcf9008d8bcd44e0ec4d05f7319133f6457c011",
```
Comment (Neumann-A) [inline versions/l-/libxmu.json:4]: overwriting this is on purpose. Otherwise the first version would be unuseable.

File: ports/libxmu/vcpkg.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "license": null,
   "dependencies": [
     "bzip2",
+    "libxext",
```
Comment (LilyWangLL) [inline ports/libxmu/vcpkg.json:9]: Thanks for your fix. Please add `"port-version": 1` for this port.

Comment (Neumann-A) [reply ports/libxmu/vcpkg.json:9]: Not bumping port-version was on purpose. see https://github.com/microsoft/vcpkg/pull/27791#discussion_r1020874258.

Comment (LilyWangLL) [reply ports/libxmu/vcpkg.json:9]: @BillyONeal Is this behavior reasonable?

Comment (BillyONeal) [reply ports/libxmu/vcpkg.json:9]: I think it should be bumped. Numbers are cheap

**Comment by BillyONeal** [comment]
Thanks!

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**